kvdb/etcd/stm.go

```diff
@@ -158,6 +172,9 @@ type stm struct {
 
 	// onCommit gets called upon commit.
 	onCommit func()
+
+	// callCount tracks the number of times we called into etcd.
+	callCount int
```

Comment:
This is to track the number of retries for a given transaction? IIRC, we were able to get this value from some other source before?

Reply:
No it's for tracking the number of times we called to `etcd` (see comment). This is used in tests to ensure we actually help with prefetches.